torchao/quantization/quant_api.py

```diff
@@ -315,7 +315,7 @@ def filter_fn(module: nn.Module, fqn: str) -> bool:
         _get_linear_subclass_inserter(apply_tensor_subclass),
         _is_linear if filter_fn is None else filter_fn,
     )
-    return model
```

Comment:
Should we still return the model? so we can support method chaining. Like the behavior of `nn.Module.to()`, `nn.Module.eval()`,

Reply:
it might be a bit weird for an inplace function to do this though?

I also thought a bit about this before, it feels like pytorch API are inplace by default? in that case we could also revert this change and leave it as is since it's consistent with other pytorch APIs.

cc @albanD do you have any thought about this question? are all pytorch APIs inplace by default?

Reply:
also there is a difference between the quantize API (a free function) v.s. `nn.Module.to/eval` (a method on an instance) I think, it does seem a bit more natural for a method to modify the state of the instance inplace, while it's a bit different for a free function to do that